bin/notification.py

```diff
@@ -6,157 +6,31 @@
 
 import json
 import logging
-from botocore.vendored import requests
```

Comment:
Remove botocore.vendored.requests in favour of standalone (in different module).